libflux/flux-core/src/semantic/types.rs

```diff
@@ -1320,7 +1328,7 @@ mod tests {
     use crate::ast::get_err_type_expression;
     use crate::parser;
     use crate::semantic::convert::convert_polytype;
-    /// Polytype is an util method that returns a PolyType from a string.
+    /// Polytype is a utility method that returns a [`PolyType`] from a string.
```

Comment:
It seems awkward to have "Polytype is a..." when the function is `polytype` and we also have a `Polytype`. Thoughts?

Reply:
I think this is alright, since it's only in the test module here. I'll change it to lowercase (`polytype`) in the doc comment on line 1331, though.